File: src/test/java/kimspring/learningtest/archunit/adapter/MyAdapter.java
package kimspring.learningtest.archunit.adapter;

import kimspring.learningtest.archunit.application.MyService2;

public class MyAdapter {
    MyService2 myService2;

    void run() {
        myService2 = new MyService2();
        System.out.println("myService2 = " + myService2);
    }
}

File: src/test/kotlin/kimspring/learningtest/archunit/adapter/MyAdapter.kt
package kimspring.learningtest.archunit.adapter

import kimspring.learningtest.archunit.application.MyService2

class MyAdapter {
    var myService2: MyService2? = null

    fun run() {
        myService2 = MyService2()
        println("myService2 = $myService2")
    }
}
